site/docs/spec.md

```diff
@@ -333,6 +333,25 @@ Table metadata is stored as JSON. Each table metadata change creates a new table
 
 The atomic operation used to commit metadata depends on how tables are tracked and is not standardized by this spec. See the sections below for examples.
 
+### Delete Format
+
+This section details how to encode row-level deletes in Iceberg metadata. Row-level deletes are not supported in the current format version, 1. This part of the spec is not yet complete and will be completed as format version 2.
```

Comment:
Can you remove the comma between version and 1?

Reply:
Sure. I force pushed the commit.